rs/execution_environment/src/execution_environment/tests.rs

```diff
@@ -2284,6 +2291,13 @@ fn make_schnorr_key(name: &str) -> MasterPublicKeyId {
     })
 }
 
+fn make_vet_kd_key(name: &str) -> MasterPublicKeyId {
```

Comment:
I'd write it as `vetkd` instead of `vet_kd` because in the management canister API it will also be written without the underscore.

For types, I think `VetKd` makes sense (with the capitalized `K`) because according to the [research paper](https://eprint.iacr.org/2023/616.pdf), the underlying cryptographic primitive is `vetKD`, and in Rust/CamelCase `VetKD` is discouraged IIUC.